catalog/java/io/material/catalog/bottomappbar/res/layout/cat_bottomappbar_fragment.xml

```diff
@@ -42,19 +42,22 @@
       app:layout_anchor="@id/bar"
       app:srcCompat="@drawable/ic_add_24px"/>
 
-  <FrameLayout
+  <RelativeLayout
       android:id="@+id/bottom_drawer"
       style="@style/Widget.Material3.BottomSheet"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      app:behavior_hideable="true"
       app:layout_behavior="@string/bottom_sheet_behavior">
-
+    <com.google.android.material.bottomsheet.BottomSheetDragHandleView
```

Comment:
Thanks for adding a drag handle here!

Reply:
I'm waiting for https://m3.material.io/components/bottom-sheets/accessibility. I'm not sure whether I should use the BottomSheetDragHandleView with the low bottom sheet.

Reply:
Yeah I think in general we encourage clients to add drag handles to all bottom sheets they have for better accessibility.

Reply:
Thank you.
I think it would be nice if the drag handle could be clicked regardless of the accessibility services...